extensions/sql-migration/src/api/utils.ts

```diff
@@ -780,17 +780,17 @@ export async function createPowershellscriptContentContainer(view: azdata.ModelV
 		},
 	}).component();
 
-	const path1 = path.join(__dirname, '../../scripts/SHIR-auto-configuration.ps1');
+	const scriptpath = path.join(__dirname, '../scripts/SHIR-auto-configuration.ps1');
 
-	const scriptContent = await fs.readFile(path1);
+	const scriptContent = await fs.readFile(scriptpath);
 
 	// inject auth keys in the script
 	const authKeys = await retrieveAuthKeys(migrationStateModel);
 	const modifiedScriptContent = await injectKeysIntoShirScriptContent
 		(authKeys.authKey1, authKeys.authKey2, scriptContent.toString());
 
 	// write it back to different file
-	const modifiedScriptPath = path.join(__dirname, '../../scripts/SHIR-auto-configuration-with-auth-keys.ps1');
+	const modifiedScriptPath = path.join(__dirname, '../scripts/SHIR-auto-configuration-with-auth-keys.ps1');
```

Comment:
That is why we should add these string constants, otherwise we need to update at everyplace. And there is risk of missing some of them.